doc/user_guide/user_guide.md

```diff
@@ -55,13 +64,13 @@ Also, we strongly recommend to change the Jupyter password as soon as possible.
 **Source**: [Github repo](https://github.com/exasol/script-languages-release)  
 The images are tightly coupled to the releases of script-languages-release; for each release of the script-languages-release there will be a release of the developer sandbox.  
 By default, the images contain a cloned repository of script-languages-container, including all dependencies to run it (Python, Poetry, Docker, etc.). The checked out version will be the tag of the respective release, e.g. a Developer Sandbox image for release 5.0.0 will have the tag 5.0.0 checked out for script-languages-release.  
-If you aim to customize existing containers, this should be fine. However, if you want to rebuild a container, you might encounter problems as dependant packages might have changed. In that case, we suggest to check out branch master  
+If you aim to customize existing containers, this should be fine. However, if you want to rebuild a container, you might encounter problems as dependant packages might have changed. In that case, we suggest to check out the master branch
 ```shell
 git checkout --recurse-submodules master
 ```
 
 For information about how to build script-languages-container please check:
-- [slc User Guide](https://github.com/exasol/script-languages-release/blob/master/doc/user_guide/user_guide.md) - Infos about the container
+- [SLC User Guide](https://github.com/exasol/script-languages-release/blob/master/doc/user_guide/user_guide.md) - Infos about the container
 - [exaslct User Guide](https://github.com/exasol/script-languages-container-tool/blob/main/doc/user_guide/user_guide.md) - Details about the build tool
```

Comment:
Should we uppercase "exaslct", too?

Reply:
We used lowercase in the script-language docu (Check https://github.com/exasol/script-languages-release/blob/master/doc/user_guide/usage.md#prerequisites).